pkg/store/foo.go

```diff
@@ -0,0 +1,102 @@
+package store
+
+import (
+	"context"
+	"errors"
+
+	"github.com/nrc-no/core/pkg/api/types"
+	uuid "github.com/satori/go.uuid"
+	"go.uber.org/zap"
+	"gorm.io/gorm"
+)
+
+// Why do I have to define this again for the migrations?
```

Comment:
You could, but then you'd have to add the `gorm` annotations on your api types, which is not so nice. Also, we have a some types that cannot be directly serialized to a flat hierarchy (such as form definitions) - so we need a different storage model

Reply:
Makes sense. I guess if my Foo (api type) and Foo (db type) differed at all then it would throw an error when trying to write to the database?

Reply:
Yeah I'm not against raw sql. orms are good for simple things (like what I'm doing here), but for complex structures they can get a bit weird.